admin/acceptance.py

```diff
@@ -365,7 +375,11 @@ def main(args, base_path, top_level):
 
     try:
         nodes = runner.start_nodes()
-        result = run_tests(nodes, options['trial-args'])
+        configure_cluster(control_node=nodes[0], agent_nodes=nodes)
+        result = run_tests(
+            nodes=nodes,
+            control_node=nodes[0], agent_nodes=nodes,
```

Comment:
Is it necessary, or meant to be obvious that the control node must be one of the agent nodes. Sorry if I'm missing some documentation, but if I'm not please make that clear if it is necessary.